airbyte-api/src/main/openapi/config.yaml

```diff
@@ -4566,6 +4574,44 @@ components:
           type: object
           additionalProperties: true
     # Web Backend
+    WebBackendConnectionListItem:
+      type: object
+      description: Information about a connection that shows up in the connection list view.
+      required:
+        - connectionId
+        - name
+        - sourceId
+        - destinationId
+        - source
+        - destination
+        - status
+        - isSyncing
+      properties:
+        connectionId:
+          $ref: "#/components/schemas/ConnectionId"
+        name:
+          type: string
+        sourceId:
+          $ref: "#/components/schemas/SourceId"
+        destinationId:
+          $ref: "#/components/schemas/DestinationId"
+        scheduleType:
+          $ref: "#/components/schemas/ConnectionScheduleType"
+        scheduleData:
+          $ref: "#/components/schemas/ConnectionScheduleData"
+        status:
```

Comment:
Is a status of inactive the same as what we call "disabled" in the frontend code?  I want to make sure we're still getting that with this call since the list view should disable the "launch" button (shows up if schedule is manual) if the connection is disabled.

Reply:
Yes, the `inactive` status is "disabled". All of these properties in the new `WebBackendConnectionListItem` are directly copied from the existing properties on `WebBackendConnectionRead`, so their usage should be exactly the same on the front-end